Assets/Scripts/API/Save/SaveTreeHeader.cs

```diff
@@ -64,25 +66,20 @@ void ReadRawData()
             MemoryStream stream = new MemoryStream(RawData);
             BinaryReader reader = new BinaryReader(stream, Encoding.UTF8);
 
-            // Read Version - must be 0x26
-            Version = reader.ReadByte();
-            if (Version != VersionNumber)
-                throw new Exception("SaveTree file has an invalid version number, must be 0x26.");
-
-            // Read CharacterPosition.RecordType - must be 0x01
-            CharacterPosition = new HeaderCharacterPositionRecord();
-            CharacterPosition.RecordType = reader.ReadByte();
-            if (CharacterPosition.RecordType != (int)RecordTypes.CharacterPosition)
-                throw new Exception("Expected CharacterPosition in SaveTreeHeader has an invalid record type, must be 0x01.");
-
-            // Read CharacterPosition.Unknown
-            CharacterPosition.Unknown = reader.ReadUInt16();
+            // Read Version - must be 0x125 or 0x126
+            Version = reader.ReadInt32();
+            if (Version < VersionNumberOld || Version > VersionNumberNew)
```

Comment:
I would prefer behaviour to match original design and support only .213 save versions. My early testing found a lot of inconsistencies in unpatched Daggerfall saves and I consider .213 to be the baseline for classic save import. Cheers. :)